aws/modules/deploy/helms/gpii-preferences-custom-values.erb

```diff
@@ -0,0 +1,19 @@
+chart-metadata:
+  name: preferences
+  namespace: gpii
+
+replicaCount: <%= ENV["TF_VAR_cluster_name"].start_with?("stg.","prd.") ? '6' : '2' %>
```

Comment:
I would prefer to get out of using ERB in favor of using Helm's templating. Is that possible?

Reply:
This is not the part of the chart, but rather a part of our rake wrapper for AWS setup.
Using Helm's templating – this was the original idea, we'll get to it once we reach that point in our GKE setup.

Reply:
Ok, makes sense. This code must stay on the AWS side only -- we'll use exekbe/terragrunt/terraform techniques on the GCP side.